yt_dlp/extractor/chzzk.py

```diff
@@ -0,0 +1,121 @@
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    traverse_obj,
+    parse_iso8601,
+)
+
+
+class CHZZKLiveIE(InfoExtractor):
+    IE_NAME = 'chzzk:live'
+    _VALID_URL = r'https?://chzzk\.naver\.com/live/(?P<id>[^/#?]+)'
```

Comment:
Channels are always hexadecimal?
```suggestion
    _VALID_URL = r'https?://chzzk\.naver\.com/live/(?P<id>[\da-f]+)'
```